### PR TITLE
Show an example of using the options

### PR DESCRIPTION
**Pull request by hansemannn**

### CODE REVIEW

**Comment by hansemannn** [comment]
I'll a note to the docs.

**Comment by hansemannn** [comment]
Updated.

File: README.md [hunk 1]
```diff
@@ -7,9 +7,20 @@ Extract and copy the contents from Android Archive (.aar) files.
...
+> Note: All paths are expected to be absolute (you can use `path.join()` for that).
+
 ```js
-var AarTransformer = require('appc-aar-transform');
-var transformer = new AarTransformer(logger) // logger is optional, can be any bunyan based logger
+var AARTransformer = require('appc-aar-transform');
+var transformer = new AARTransformer(logger) // logger is optional, can be any bunyan based logger
+
+var options = {
+  aarPathAndFilename: 'path/to/file.aar',
```
Comment (janvennemann) [inline README.md]: Please add a leading slash so its obvious this and the following options are expected to be absolute paths.

**Comment by hansemannn** [comment]
@janvennemann Updated 🚀

**Pull request merged**